plugin/types.go

```diff
@@ -193,3 +193,11 @@ type Readable interface {
 	Entry
 	Open(context.Context) (SizedReader, error)
 }
+
+// Permissible is used to define how to check that adequate permissions to access the resource are
```

Comment:
To make this work with external plugins we'd need a Permitted method to check. Obviously that could be statically declared when listing the items.

Reply:
An attribute will be simpler if we're getting rid of the need to test whether an entry is Permissible (or making it an explicit thing a plugin needs to set).